backend/core/tests/test_admin_functionality_make_team_members_inactive.py

```diff
@@ -0,0 +1,42 @@
+from core.tests import factories
+from django.test import TestCase
+from core.admin import TeamAdmin
+from mock import patch
+
+
+class TestMakeTeamMembersInactiveForATeamOrTeams(TestCase):
```

Comment:
Is the `OrTeams` necessary? Can the operation be done for multiple teams simultaneously?